Application.UnitTests/MovieEnthusiast.Application.UnitTests.csproj

```diff
@@ -0,0 +1,32 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net7.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+
```

Comment:
Let's be consistent across all projects and add the following inside each < Project > node (same for other newly added projects)
```
  <PropertyGroup>
    <TreatWarningsAsErrors>True</TreatWarningsAsErrors>
    <WarningLevel>9999</WarningLevel>
    <AnalysisLevel>preview</AnalysisLevel>
  </PropertyGroup>
```